shared/menubar/dumb.desktop.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import Menubar from './index.render'
+import type {RenderProps, FolderInfo} from './index.render'
+import type {DumbComponentMap} from '../constants/types/more'
+
+const folder1: FolderInfo = {
+  type: 'folder',
+  folderName: 'max,chris',
+  isPublic: true,
+  isEmpty: false,
+  openFolder: () => {}
+}
+
+const folderLong: FolderInfo = {
+  type: 'folder',
+  folderName: 'max,chris,marcopolo,patrick,strib,mgood,zanderz,gabrielh,chrisnojima,cbostrander,alness,akalin',
+  isPublic: true,
+  isEmpty: false,
+  openFolder: () => {}
+}
+
+const folder2: FolderInfo = {
+  type: 'folder',
+  folderName: 'mgood,strib,marcopolo',
+  isPublic: true,
+  isEmpty: false,
+  openFolder: () => {}
+}
+
+const propsNormal: RenderProps = {
```

Comment:
It won't unless you change https://github.com/keybase/client/blob/master/shared/menubar/index.render.js.flow#L20 to be `export default class Render extends Component<void, RenderProps, void> { }`